themes/default/content/blog/iac-recommended-practices-wrapping-up/index.md

```diff
@@ -0,0 +1,81 @@
+* Zephyr quickly added per-developer stacks as a way to enhance developer productivity. _(You can read about per-developer stacks in [the second post](/blog/iac-recommended-practices-developer-stacks-git-branches/) in the series.)_
+* As Zephyr continued to grow, their single Pulumi project grew into three different Pulumi projects: one for base infrastructure, one for their Kubernetes platform, and one for the online store application. _(The [third post in the series](/blog/iac-recommended-practices-structuring-pulumi-projects/) describes the reasoning for adopting multiple projects.)_
+* Accompanying the switch to multiple projects, Zephyr implemented stack references (to pass information between stacks in different projects) and applied role-based access control (RBAC) to the stacks in Pulumi Cloud. _(Refer back to [the fourth post](/blog/iac-recommended-practices-using-stack-references/) and [the fifth post](/blog/iac-recommended-practices-rbac-and-security/) in the series for more details on each of these areas.)_
+* Zephyr realized a need to split out their data layer, going from three projects to four projects, and along the way saw an opportunity to add a higher level of orchestration using the Pulumi Automation API. _(Refer back to [our sixth post](/blog/iac-recommended-practices-using-automation-api/) for more details on Zephyr's initial use of Automation API.)_
+
+Since the last blog post, the Zephyr team has continued to explore Automation API, and has added an Automation API program that allows their developers to deploy the Zephyr online store directly from the associated GitHub repositories---developers don't even need to clone down the repositories first.
+
+{{% notes type="info" %}}
+To see the latest Automation API programs for Zephyr, refer to [the `zephyr-automation` GitHub repository](https://github.com/pulumi-zephyr/zephyr-automation).
+{{% /notes %}}
+
+## Areas of future growth for Zephyr
+
+We've stated before that recommended practices (or even "best practices," if you prefer that term) are always point-in-time recommendations based on the requirements of the organization/team/individual using Pulumi. What this means, in practice, is that there is almost always room for growth and evolution. Organizations, teams, and individuals rarely remain static, and therefore their use of Pulumi is also unlikely to remain entirely static.
+
+So, what are some potential areas of future growth for Zephyr?
+
+* **Splitting the deployment of the microservices**: We alluded to this [in the previous post](/blog/iac-recommended-practices-using-automation-api/). Right now, all the microservices that comprise the online store are deployed together. What if one particular team needs to deploy only their service? The current approach can certainly accommodate that---Pulumi is declarative and will only change what needs to be changed---but would it be better to split this into separate Pulumi projects?
```

Comment:
Is it possible to tie the 'would it be better' to the 'what's better?' is this decision based on flexibility for teams to move more quickly, reduce the likelihood of some sort of unintended change (limit blast radius?) Mirror the team/org design?

Reply:
I've expanded on this a bit; let me know what you think.